app/feature/feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/data/GetContractsToTerminateUseCase.kt

```diff
@@ -0,0 +1,63 @@
+package com.hedvig.android.feature.terminateinsurance.data
+
+import arrow.core.Either
+import arrow.core.raise.either
+import com.apollographql.apollo3.ApolloClient
+import com.hedvig.android.apollo.safeExecute
+import com.hedvig.android.apollo.toEither
+import com.hedvig.android.core.common.ErrorMessage
+import com.hedvig.android.core.uidata.UiMoney
+import com.hedvig.android.data.contract.ContractGroup
+import com.hedvig.android.data.contract.toContractGroup
+import com.hedvig.android.logger.logcat
+import kotlinx.datetime.LocalDate
+import octopus.ContractsToTerminateQuery
+
+internal interface GetContractsToTerminateUseCase {
```

Comment:
This is potentially the same piece of logic that the help-center may need to reuse in order to know when not to show the termination flow.